ifrs17/Constants/Enums.ipynb

```diff
@@ -299,7 +299,13 @@
         {
             "cell_type": "markdown",
             "source": [
-                "## Import Scope"
+                "## Import Scope",
+                "\nImport Scope is used in the [ImportStorage](../Import/ImportStorage) and in the [Compute Function](../Import/Importers#calculate-ifrs-variables-for-all-scenarios) to drive the calculation of new IfrsVariables. Each import scope contains a collection of Data Node.",
+                "\n",
+                "\nIts values indicate: ",
+                "\n- <code>Primary</code> : Data nodes for which [IfrsVariable](../DataModel/DataStructure#ifrs-variable) will be calculated.",
+                "\n- <code>AddedToPrimary</code> : Subset of Primary. It contains data nodes linked to data nodes provided in input that need recalculation. ",
+                "\n- <code>Secondary</code> : Data nodes that provide inputs to the Data nodes in primary scope. No variable will not be calculated."
```

Comment:
@dcolleoni in the markdown for secondary scope, do you instead mean "No new variable will be calculated"?